.github/workflows/regressiontest_container.yml

```diff
@@ -0,0 +1,35 @@
+name: Regression Test in Container
+
+on:
+  pull_request:
+    branches: [ devel ]
+    
+jobs:
+  buildtest_tutorial_container:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/buildtesters/buildtest_spack
```

Comment:
can you try `ghcr.io/buildtesters/buildtest_spack:spack-sc23` for whatever reason its not finding `csh` and it should. You can try running `which csh zsh` prior to running regtest for debugging purposes

Reply:
its in the container i just checked see this line ihttps://github.com/buildtesters/buildtest/actions/runs/9692887989/job/26747175463#step:4:17

Reply:
its here

<img width="320" alt="Screenshot 2024-06-28 at 11 50 47 AM" src="https://github.com/buildtesters/buildtest/assets/12942230/e57dc5f8-b218-4ae1-9c22-97b1935541e2">

Reply:
I know, but the test failed: https://github.com/buildtesters/buildtest/actions/runs/9692887989/job/26747175463#step:4:164

Reply:
ur right this is out of scope we will need to fix this for future. I will merge this in.

It appears that `/usr/bin/csh` is not available shell for whatever reason we are `cat /etc/shells` which contains a list of shell. Dont know why its installed as `/usr/bin/csh` there is typically a symlink to `/bin/csh`.